src/components/file-handler/FileHandler.stories.mdx

```diff
@@ -0,0 +1,107 @@
+import {ArgsTable, Meta, Story, Canvas} from '@storybook/addon-docs';
+import {StoryVariant} from '../../_docs/utils';
+import {Preview} from '../../_docs/blocks/';
+import FileHandler, {COLORS_MAP} from './FileHandler';
+import {TYPE} from '../icons/Icon';
+import FileHandlerA11y from './stories/FileHandler.a11y.mdx';
+import Flex from '../flex/Flex';
+const src =
+  // eslint-disable-next-line max-len
+  'https://images.unsplash.com/photo-1558349699-1e1c38c05eeb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=240&q=80';
+
+<Meta
+  title="Components/FileHandler"
+  component={FileHandler}
+  argTypes={{
+    children: {
+      control: {
+        disable: true,
+      },
+    },
+    textRef: {
+      control: {
+        disable: true,
+      },
+    },
+    iconType: {
+      control: {
+        type: 'select',
+        options: TYPE,
+      },
+    },
+  }}
+  args={{
+    children: 'avatar.jpg',
+  }}
+/>
+
+# FileHandler
+
+- [Stories](#stories)
+- [Accessibility](#accessibility)
+
+## Overview
+
+<Canvas>
+  <Story name="Default">{args => <FileHandler {...args} />}</Story>
+</Canvas>
+
+<ArgsTable story="Default" />
+
+## Stories
+
+### Colors
+
+<Canvas>
+  <Story name="Colors">
+    {args => (
+      <div>
+        {Object.values(COLORS_MAP).map(color => (
+          <StoryVariant label={`color - ${color}`} key={color}>
+            {color === 'white' ? (
+              <div className="sg-story-variant-dark-box">
+                <FileHandler {...args} color={color} />
+              </div>
+            ) : (
+              <FileHandler {...args} color={color} />
+            )}
+          </StoryVariant>
+        ))}
+      </div>
+    )}
+  </Story>
+</Canvas>
+
+### Icons
+
+<Canvas>
+  <Story name="Icons">
+    {args => (
+      <Flex wrap>
+        {Object.values(TYPE).map(icon => (
+          <StoryVariant label={`icon - ${icon}`} width={200} key={icon}>
+            <FileHandler {...args} iconType={icon} />
+          </StoryVariant>
+        ))}
+      </Flex>
+    )}
+  </Story>
+</Canvas>
+
+### With thumbnail
+
+<Canvas>
+  <Story name="With thumbnail">
+    {args => <FileHandler {...args} thumbnailSrc={src} />}
```

Comment:
Can we rather try with such approach?:
```
<Story name="With thumbnail" args={{thumbnailSrc: src}}>
    {args => <FileHandler {...args} />}
```
What do you think? I followed such convention, so I guess we should decide to stick to one approach
@coderitual @clxandstuff

Reply:
I just checked how args setup affects the story itself and I think we should follow my approach. See this example from Dropdown:
https://github.com/brainly/style-guide/pull/2384/files#diff-7cb061b0877f860733a6f0d92ad01130d44d2b48e2f3ad7bbfaeb446e3f27ec8R82

When an arg is being set on the Story, rather than component, it renders controls properly!
![Screenshot 2022-04-13 at 16 38 36](https://user-images.githubusercontent.com/12099980/163205626-e32d5948-6f91-4061-981d-046c4483bc76.png)

Reply:
nice catch!

Reply:
@domidomi @katarzynatobis I wouldn't say this is a strict requirement for stories different than default. We have many stories where changing args don't make a lot of sense (e.g. in buttons) but we can agree that we can pass via args if it makes sense.